data/labs_data.json

```diff
@@ -62,8 +62,8 @@
             "answer": "Yes! ADI Labs is structured to help you achieve your goals and finish a real project. If you already have one in mind, let us know what your idea is in the application. Make sure to flesh out your idea and how you can accomplish it; the more detail the better! However, it is not guaranteed you will get to work on the project you propose."
         },
         {
-            "question": "I don’t program, but I’m interested in design, product management or marketing. Can I also get involved?",
-            "answer": "Yes!  When you apply, just indicate in which roles you’d see yourself.  We’ll work with you to place you on a team where you can be most effective!"
+            "question": "I don’t program, but I’m interested in design and/or product. Can I also get involved?",
```

Comment:
since the product management role was removed, why does this question still include product